frontend/src/bundles/employers/pages/candidates.tsx

```diff
@@ -53,10 +51,14 @@ const Candidates: React.FC = () => {
         useAppForm<EmployeesFiltersDto>({
             defaultValues: DEFAULT_EMPLOYEES_FILTERS_PAYLOAD,
```

Comment:
when navigating to another page and then return - filters are reset, I mean if I go to the candidate page and then back to search I have all filters default
try to use [this changes](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/625/commits/687988e906e076eafa66a1aee0c01f49119d65e5) from draft pr 
here they are not reset but then got a problem with clearing filters to the default with Clear filters button

Reply:
actually fixed _Clear filters handling_ in [this commit](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/625/commits/0b5cd42d558c01c61f1cce4d09ee2c16d1565d46)
So you can changes from these 2 commits to your PR

Reply:
Added